### ISSUE TITLE
Add prop for a main app container that should have aria-hidden=true when modal is open

### ISSUE THREAD

**Issue by davidtheclark** [davidtheclark/react-aria-modal#10]
Per: https://www.w3.org/WAI/GL/wiki/Using_ARIA_role%3Ddialog_to_implement_a_modal_dialog_box#Description
